requirements.txt

```diff
@@ -10,11 +10,13 @@ plotly==5.14.0
 pythreejs
 pyvista>=0.43.0
 ipyreact>=0.4.1
+requests
 trame
 trame-vuetify
 trame-vtk
 tvb-library
 tvb-framework
+tvb-ext-bucket
 plotly-resampler

```

Comment:
Maybe here we need to constraint to `plotly-resampler<=0.9.2` ?
In my recent tests the latets version of resamples seems not to work (python 3.11). Could you pls test (maybe even another PR).

Reply:
On my machine I tested with latest version(=0.10) of plotly-resampler and python 3.11.9 and it works

Reply:
I also can not reproduce the problem any more. It might have been a broken installation when I had the import issue. Thanks for checking.